app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -519,6 +525,21 @@ class MainViewModel(application: Application) : AndroidViewModel(application) {
         }
     }
 
+    fun reportPhishing(threadUid: String, message: Message) = viewModelScope.launch(Dispatchers.IO) {
+        val mailboxUuid = currentMailbox.value?.uuid ?: return@launch
+
+        val apiResponse = ApiRepository.reportPhishing(mailboxUuid, message.folderId, message.shortUid)
+
+        val snackbarTitle = if (apiResponse.isSuccess()) {
+            if (!isCurrentFolderRole(FolderRole.SPAM)) toggleSpamOrHam(threadUid, message, false)
+            R.string.snackbarReportPhishingConfirmation
+        } else {
+            RCore.string.anErrorHasOccurred
+        }
+
```

Comment:
There is a `refreshFolders()` call missing here.

Reply:
Erratum: Nothing is missing here.
The phishing action isn't moving any Message, so there isn't any need to refresh Folders.